### PR TITLE
Implemented Movie Search by Title

### PR DESCRIPTION
**Pull request by ahadinger**

### CODE REVIEW

**Comment by AlexStahlman** [comment]
may want to remove "movie_repository_singleton.create_movie("Land Before Time", "idk", 9.0)" in like 33 of app.py

**Comment by ahadinger** [comment]
Somehow missed my e2e tests on my first set of commits, should be fixed now

File: app.py [hunk 1]
```diff
@@ -29,5 +29,15 @@ def create_movie():
...
-    # TODO: Feature 3
-    return render_template('search_movies.html', search_active=True)
+    title = request.args.get("title")
+    movie = movie_repository_singleton.get_movie_by_title(title)
+    rating = 0
+    if movie != None:
+        rating = movie.rating
+
+    if movie != None: #movie exists and title is valid
+        return render_template('search_movies.html', search_active=True, output = 1, _title = title, _rating = rating)
```
Comment (DylanHalstead) [inline app.py:39]: Not sure what exactly _title = title and _rating = rating exactly does, but looks good!

**Review by DylanHalstead — APPROVED** [review]
LGTM

**Pull request merged**